app/(gcforms)/[locale]/(form administration)/form-builder/components/shared/conditionals/AddOther.tsx

```diff
@@ -46,16 +58,25 @@ export const AddOther = ({ item }: { item: FormElementWithIndex }) => {
     };
 
     const allowGroups = await allowGrouping();
+    let itemId = 0;
     if (allowGroups) {
-      add(item.index, FormElementTypes.textField, data, groupId);
+      itemId = await add(item.index, FormElementTypes.textField, data, groupId);
     } else {
-      add(item.index, FormElementTypes.textField, data);
+      itemId = await add(item.index, FormElementTypes.textField, data);
     }
+
+    const newRule = { elementId: `${itemId}`, choiceId: `${item.id}.${lastChoice}` };
+    OnComplete(newRule);
   }, [add, item, groupId]);
```

Comment:
See lint error here

Reply:
I'm not getting any lint error flagged? What are you seeing?

Reply:
https://github.com/cds-snc/platform-forms-client/pull/4002/files#diff-0087593d121005500352fe89851cd5469758f2016edaef5a81a0beda6343995eR69